dev/com.ibm.ws.microprofile.metrics.3.0/src/com/ibm/ws/microprofile/metrics30/helper/Util30.java

```diff
@@ -0,0 +1,150 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.metrics30.helper;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.SortedSet;
+
+import org.eclipse.microprofile.metrics.Metadata;
+import org.eclipse.microprofile.metrics.Metric;
+import org.eclipse.microprofile.metrics.MetricID;
+import org.eclipse.microprofile.metrics.MetricRegistry;
+import org.eclipse.microprofile.metrics.SimpleTimer;
+import org.eclipse.microprofile.metrics.Timer;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.ws.microprofile.metrics.Constants;
+import com.ibm.ws.microprofile.metrics.exceptions.EmptyRegistryException;
+import com.ibm.ws.microprofile.metrics.exceptions.NoSuchMetricException;
+import com.ibm.ws.microprofile.metrics.exceptions.NoSuchRegistryException;
+import com.ibm.ws.microprofile.metrics23.helper.Util23;
+
+/**
+ *
+ */
+public class Util30 extends Util23 {
+    private static final TraceComponent tc = Tr.register(Util30.class);
+
+    public static Map<MetricID, Metric> getMetricsAsMap(String registryName, String metricName) throws NoSuchRegistryException, NoSuchMetricException, EmptyRegistryException {
```

Comment:
Do you expect FFDCs for the thrown exceptions?

Reply:
Exception is caught later and an FFDC ignore is applied there.